cli/cmd/testdata/validate-config-error.yml

```diff
@@ -0,0 +1,20 @@
+kind: source
+spec:
+  name: aws
+  path: cloudquery/aws
+  registry: cloudquery
+  version: "v25.5.0"
```

Comment:
Our CLI tests are unauthenticated so using this version might not work

Reply:
I am seeing other weird issues as well even locally

Reply:
https://github.com/cloudquery/cloudquery/pull/17530/commits/38c85d8443bffe1f6153bea3c2e4df029563663a should fix it and uses `cloudflare` as it's faster to download